manifests/testing/cdi-v1.5.yaml.in

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    cdi.kubevirt.io: ""
+  name: {{.CDINamespace}}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cdi-operator
+  namespace: {{ .Namespace }}
+  labels:
+    operator.cdi.kubevirt.io: ""
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cdi-operator
+  labels:
+    operator.cdi.kubevirt.io: ""
+subjects:
+- kind: ServiceAccount
+  name: cdi-operator
+  namespace: {{ .Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "cdi-controler-leader-election"
+  namespace: {{ .Namespace }}
+  labels:
+    operator.cdi.kubevirt.io: ""
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cdi-operator
+  namespace: {{ .Namespace }}
+  labels:
+    operator.cdi.kubevirt.io: ""
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: cdi-operator
+  template:
+    metadata:
+      labels:
+        name: cdi-operator
+    spec:
+      serviceAccountName: cdi-operator
+      containers:
+        - name: cdi-operator
+          image: {{ .DockerRepo }}/{{ .OperatorImage }}:{{ .DockerTag }}
+          ports:
+          - containerPort: 60000
+            name: metrics
+          imagePullPolicy: Always
+          env:
+          - name: DEPLOY_CLUSTER_RESOURCES
+            value: 'true'
+          - name: DOCKER_REPO
+            value: {{ .DockerRepo }}
+          - name: DOCKER_TAG
+            value: {{ .DockerTag }}
+          - name: CONTROLLER_IMAGE
+            value: {{ .ControllerImage }}
+          - name: IMPORTER_IMAGE
+            value: {{ .ImporterImage }}
+          - name: CLONER_IMAGE
+            value: {{ .ClonerImage }}
+          - name: APISERVER_IMAGE
+            value: {{ .APIServerImage }}
+          - name: UPLOAD_PROXY_IMAGE
+            value: {{ .UploadProxyImage }}
+          - name: UPLOAD_SERVER_IMAGE
+            value: {{ .UploadServerImage }}
+          - name: VERBOSITY
+            value: '{{ .Verbosity }}'
+          - name: PULL_POLICY
+            value: {{ .PullPolicy }}
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
Somewhere, an instance of this CRD has to be created or CDI won't really be installed.  See:  https://github.com/kubevirt/containerized-data-importer/blob/master/manifests/templates/cdi-operator-cr.yaml.in

Can add it to this manifest or make it's own.  But just have to make sure that the instance is created after the operator is installed.

Reply:
Yeah, I am having an issue with that right now, if I drop in the cr in the same directory, the operator won't be operational when it gets called and it will fail.

Reply:
Why not just add it in this file?

Reply:
Oh, you did.